extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/vertx/VertxHttpServerMetrics.java

```diff
@@ -144,6 +145,8 @@ public void responseEnd(HttpRequestMetric requestMetric, HttpResponse response,
         if (path != null) {
             Timer.Sample sample = requestMetric.getSample();
             Timer.Builder builder = Timer.builder(nameHttpServerRequests)
+                    .minimumExpectedValue(Duration.ofNanos(1L))
+                    .maximumExpectedValue(Duration.ofNanos(Double.valueOf(Double.POSITIVE_INFINITY).longValue()))
                     .tags(Tags.of(
```

Comment:
I know that the exemplar technically needs this.. but exemplars are not used unless histograms are enabled, and that should be done by the user when they want this data. I'm not sure this min/max helps otherwise (it should be set by the user based on their expectations..).

Reply:
I do agree, I opened an issue upstream : https://github.com/micrometer-metrics/micrometer/issues/3485

They seems to agree min/max should contain the exemplar, but the spec doesn't agree...